### PR TITLE
Finish lambda exercise

### PR DESCRIPTION
**Pull request by shaharshw**

### CODE REVIEW

File: src/main/kotlin/com/hibob/lambda/lambda.kt [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+            appendLine("Title: $title")
+            appendLine("Air date: $airDate")
+            appendLine("IMDB rank: $imdbRank")
+        }
+
+        return printerMovie
+    }
+
+    override fun signTobeyMaguire() {
+        //  Tobey signed!
```
Comment (yaelblsch) [inline src/main/kotlin/com/hibob/lambda/lambda.kt:63]: You can add some implementation here

File: src/main/kotlin/com/hibob/lambda/lambda.kt [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+    fun init(): Boolean {
+
+        return movieProducer?.run {
+            if (!isThereLockdown()) {
+                signTobeyMaguire()
+                signAndrew()
+                signTom()
+                getVillains()
+                publish()
+                true
```
Comment (yaelblsch) [inline src/main/kotlin/com/hibob/lambda/lambda.kt]: do you really need to return true here?

File: src/main/kotlin/com/hibob/lambda/lambda.kt [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+            }
+            else false
+        } ?: false
```
Comment (yaelblsch) [inline src/main/kotlin/com/hibob/lambda/lambda.kt:125]: you solution here is good - just note that you can also move the return to within the run before the if and use return false at the end without ?:
Just another way of wiritng

File: src/main/kotlin/com/hibob/lambda/lambda.kt (hunk 2 above)
Comment (shaharshw) [reply src/main/kotlin/com/hibob/lambda/lambda.kt]: thanks i changed it.

**Review by yaelblsch — APPROVED** [review]

**Pull request merged**